src/components/Image/index.tsx

```diff
@@ -0,0 +1,14 @@
+import NextImage, { ImageProps } from 'next/image';
+
+const Image = ({ src, children, ...rest }: ImageProps) => {
+  const isProd = process.env.NODE_ENV === 'production';
+  const finalSrc = isProd ? src : `${process.env.NEXT_PUBLIC_BASE_PATH}${src}`;
+
+  return (
+    <NextImage src={finalSrc} data-test-id="image-wrapper" {...rest}>
```

Comment:
Maybe we could spare the responsibility of defining the `data-test-id` to the calling component? The reason is, that makes testing the image `src` a lot easier:

```javascript
    const props = {
      src: '/image.svg',
      alt: 'image',
      'data-test-id': 'image',
    };

    render(<Image {...props} />);

    const image = screen.getByTestId(props['data-test-id']);

    expect(image).toBeVisible();
    expect(image).toHaveAttribute('src', props.src);
```